multicore_runtime/wsgi.py

```diff
@@ -23,6 +23,7 @@
 import logging
 import os
 
+from cloud_logging import CloudLoggingHandler
```

Comment:
Google style guide doesn't allow importing class directly. Not sure whether that applies here or not.